sig-etcd/README.md

```diff
@@ -49,6 +49,12 @@ subprojects, and resolve cross-subproject technical issues and decisions.
     - [@kubernetes/sig-etcd-leads](https://github.com/orgs/kubernetes/teams/sig-etcd-leads) - SIG Chairs and Tech Leads
 - Steering Committee Liaison: Bob Killen (**[@mrbobbytables](https://github.com/mrbobbytables)**)
 
+## Working Groups
+
+The following [working groups][working-group-definition] are sponsored by sig-etcd:
+* [WG etcd Operator](/wg-etcd-operator)
+
+
 ## Subprojects
```

Comment:
Probably, we can add the etcd operator subproject info here as this is in scope of the wg.

Reply:
thx for the reminder. 
The subproject `etcd-io/etcd-operator` hasn't been created yet. Will update here once it's created in future.